### PR TITLE
OCPBUGS#11250: Add prohibited tags for AWS VPC

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.11+

Issue: https://issues.redhat.com/browse/OCPBUGS-11250

Link to docs preview: https://58637--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-vpc.html#installation-custom-aws-vpc-requirements_installing-aws-vpc

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-custom-aws-vpc.adoc [hunk 1]
```diff
@@ -96,7 +96,7 @@ In case you need to create a public subnet in the AWS Outposts, verify that this
 To access your local cluster over your local network, the VPC must be associated with your Outpost's local gateway route table. For more information, see link:https://docs.aws.amazon.com/outposts/latest/userguide/outposts-local-gateways.html#vpc-associations[VPC associations] in the AWS Outposts User Guide.
 ====
 endif::aws-outposts[]
-* The VPC must not use the `kubernetes.io/cluster/.*: owned` tag.
+* The VPC must not use the `kubernetes.io/cluster/.*: owned`, `Name`, and `openshift.io/cluster` tags.
```
Comment (shaozha) [inline modules/installation-custom-aws-vpc.adoc:99]: @xenolinux Hi, I suggest explaining the reason why 'Name' cannot be used at the bottom of this line, similar to the description of 'kubernetes. io/cluster/. *: shared' tag in line 101 of the document.

Comment (xenolinux) [reply modules/installation-custom-aws-vpc.adoc:99]: @shaozha I updated the PR- can you PTAL?

File: modules/installation-custom-aws-vpc.adoc [hunk 2]
```diff
@@ -96,9 +96,9 @@ In case you need to create a public subnet in the AWS Outposts, verify that this
...
 +
-The installation program modifies your subnets to add the `kubernetes.io/cluster/.*: shared` tag, so your subnets must have at least one free tag slot available for it. See link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html#tag-restrictions[Tag Restrictions] in the AWS documentation to confirm that the installation program can add a tag to each subnet that you specify.
+The installation program modifies your subnets to add the `kubernetes.io/cluster/.*: shared` tag, so your subnets must have at least one free tag slot available for it. See link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html#tag-restrictions[Tag Restrictions] in the AWS documentation to confirm that the installation program can add a tag to each subnet that you specify. You cannot use a `Name` tag, because it overlaps with the EC2 `Name` field and installation fails.
```
Comment (mburke5678) [inline modules/installation-custom-aws-vpc.adoc:101]: s/installation fails/the installation fails

**Comment by mburke5678** [comment]
@xenolinux One tiny nit. Otherwise LGTM

**Comment by michaelryanpeter** [comment]
@xenolinux The cherrypick for 4.11 failed. Would you please perform a manual cherrypick? Thanks!

**Pull request merged**